oeq-ts-rest-api/src/index.ts

```diff
@@ -35,3 +35,4 @@ export * as Security from './Security';
 export * as Settings from './Settings';
 export * as UserQuery from './UserQuery';
 export * as Utils from './Utils';
+export * as WizardControl from './WizardControl';
```

Comment:
While here, could you also:

```suggestion
export * as WizardCommonTypes from './WizardCommonTypes';
export * as WizardControl from './WizardControl';
```

As I've needed to do that in my branch, so can I assure you it is needed. :wink: